io/FV3GFS_io.F90

```diff
@@ -962,11 +962,11 @@ subroutine sfc_prop_restart_read (Sfcprop, Atm_block, Model, fv_domain)
     endif
 #endif
       !--- register the 3D fields
-    if (Model%frac_grid) then
+!   if (Model%frac_grid) then
```

Comment:
Are these if tests ever going to be uncommented again. If not, consider deleting them.

Reply:
Not sure, I'd prefer Moorthi or Shan to do that with one of their next commits.